keyboards/xd75/keymaps/skewwhiffy/keymap.c

```diff
@@ -15,36 +15,44 @@
  */
 #include "xd75.h"
```

Comment:
Please change Line 16 to `#include QMK_KEYBOARD_H`.

Reply:
Done.